src/interfaces/buttonGroup.d.ts

```diff
@@ -0,0 +1,17 @@
+// <reference types="react" />
+import { ButtonSize } from "./buttons";
+
+export interface IButtonGroupProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  fullWidth?: boolean;
+  className?: string;
+  currentIndex?: number;
+  size: ButtonSize;
+  children: React.ReactNode;
+  onIndexChange?: (
+    event: React.MouseEvent<HTMLDivElement>,
```

Comment:
should `event` be here? :> You have `this.props.onIndexChange(index)` in the code

Reply:
IMHO it a good thing to pass `event`. Please change it in the component

Reply:
🙈

Reply:
ok, updated 👍